engine/class_modules/warlock/sc_warlock_demonology.cpp

```diff
@@ -302,7 +302,10 @@ struct demonbolt_t : public demonology_spell_t
 
     p()->buffs.balespiders_burning_core->expire();
 
-    p()->buffs.decimating_bolt->decrement();
+    if ( rng().roll( p()->conduit.prolonged_decimation.percent() ) )
```

Comment:
Similar to above, this needs moving to impact()

Reply:
Done!